hack/local-development/local-garden/run-kube-apiserver

```diff
@@ -10,19 +10,48 @@ if ! ( [[ "$(uname -s)" == *"Darwin"* ]] || [[ "$(uname -s)" == "Linux" && "$(un
 fi
 
 LABEL=${1:-local-garden}
-
-SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+ACTIVATE_SEEDAUTHORIZER=${2:-false}
 
 IMAGE=k8s.gcr.io/kube-apiserver:v1.20.2
-MOUNTS="-v $SCRIPTPATH/certificates/certs:/certs -v $SCRIPTPATH/certificates/keys:/keys"
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+MOUNTS="-v $SCRIPTPATH/certificates/certs:/certs -v $SCRIPTPATH/certificates/keys:/keys -v $SCRIPTPATH/configs:/configs"
 ETCD_PORT=12379
 LISTEN_PORT="2443"
 
+ADMISSION_CONTROLLER_EXTERNAL_NAME=gardener.localhost
+if [[ "$(uname -s)" == *"Darwin"* ]] || [[ "$(uname -s)" == "Linux" && "$(uname -r)" =~ "microsoft-standard" ]] ; then
+  ADMISSION_CONTROLLER_EXTERNAL_NAME=host.docker.internal
+fi
+
+cat<< EOF > "$SCRIPTPATH/configs/authwebhook.yaml"
```

Comment:
Can we create this in the `dev` dir?

Reply:
We can, but I avoided this to not "smear" the different configuration for the local kube-apiserver in too many places. I think it's better to have everything in one place.

Reply:
Hm, to me the `dev` dir was always such a place. Everything that is generated, configuration related or data for development lands there and is ignored by Git i.e., no additional `.gitignore` and once place to go to change configuration. The etcd data dir for our local K8s is also already stored in  `dev`.

I think the certs and kubeconfigs are only inside the `local-garden` dir to check them in which we ought to get rid of in the future.

WDYT?

Reply:
I'm fine to either have everything in `hack` or everything in `dev`, but not scattered around both places (OK, it's partly already like this, because the etcd data dir is in `dev`, however, I don't like this and don't want to make it worse). So not sure, I'm still not convinced. Even if the target picture is to have everything (including the certs, etc.) in `dev` then I would still vote to keep it as it is for now and rather refactor it consistently with this subsequent endeavour.

Also, (at least that's my understanding) `dev` was more meant to contain files that the developer might want to tweak (e.g., component configuration or shoot examples). This auth configuration is not of this kind, it's static and it's highly unlikely that a developer ever needs to touch or even care about it. It's just that there is one setting that is depending on the operating system, so it can't be "hard-coded" like the other files for the local garden. Anyways, that's just a minor detail and not too relevant - my main motivation is consistency.

Reply:
> my main motivation is consistency

Mine is too and since we've never defined or discussed the structure, the way I described it was the way I understood it (`dev` == generated config files for dev, ignored by Git, `hack` == static and checked-in assets). 

> Also, (at least that's my understanding) dev was more meant to contain files that the developer might want to tweak

Okay then, let's follow your definition then.

Reply:
Yeah, let's continue for now as this is minor with little impact anyways. If you already have ideas how to improve it then it can be done consistently in the future. 👍🏻